app/templates/events/view/sessions.hbs

```diff
@@ -0,0 +1,103 @@
+<div class="ui basic buttons">
+  <button class="ui button">All</button>
+  <button class="ui button">Pending</button>
+  <button class="ui button">Accepted</button>
+  <button class="ui button">Confirmed</button>
+  <button class="ui button">Rejected</button>
+</div>
+<button class="ui teal button">Create Session</button>
+<select class="ui dropdown">
```

Comment:
In my opinion drop down icon will look good :)
![screenshot from 2017-05-26 18 36 36](https://cloud.githubusercontent.com/assets/12701036/26495750/69f67cba-4242-11e7-9b00-727f7344cfe2.png)